platformio.ini

```diff
@@ -121,7 +121,7 @@ build_flags   = -DLEDSTRIP=1
 ; so it sends power and ground as well as data on PIN 32.D
 
 [env:m5demo]
-platform      = espressif32wn
+platform      = espressif32@3.5.0
```

Comment:
I noticed that the original platform identifier here is espressif32**wn**, instead of espressif32. Have you established that the "non-wn" version works in this context?

Reply:
I get the error, and the same error on windows
Platform Manager: Installing espressif32wn
Error: Could not find the package with 'espressif32wn' requirements for your system 'linux_x86_64'
I googled this package and the only result is this one, so it seemed to be a typo. 
Another change is needed to compile a build flag is missing from env:m5demo
	-DM5STICKC=1
I'll create another PR or edit this one when I work out what to do.

Reply:
Ok, thank you for the explanation. I'll proceed with merging this PR as it is now, because as it stands the 4.x version of espressif32 breaks builds that were previously working.